rustfmt.toml

```diff
@@ -1,275 +1,7 @@
-# Values: WithArgs | WithWhereClause
-# Default: WithArgs
-fn_return_indent = "WithArgs"
-
-# If function argument parenthesis goes on a newline.
-# Default: true
-# fn_args_paren_newline =
-
-# Argument density in functions.
-# Values: Compressed | Tall | CompressedIfEmpty | Vertical
-# Default: Tall
-fn_args_density = "Tall"
-
-# Layout of function arguments.
-# Values: Visual | Block | BlockAlways
-# Default: Visual
-fn_args_layout = "Visual"
-
-# Indent on function arguments.
-# Values: Inherit | Tabbed | Visual
-# Default: Visual
-fn_arg_indent = "Visual"
-
-# Determines if '+' or '=' are wrapped in spaces in the punctuation of types.
-# Values: Compressed | Wide
-# Default: Wide
-type_punctuation_density = "Wide"
-
-# Density of a where clause.
-# Values: Compressed | Tall | CompressedIfEmpty | Vertical
-# Default: CompressedIfEmpty
-# where_density =
-
-# Indentation of a where clause.
-# Values: Inherit | Tabbed | Visual
-# Default: Tabbed
-where_indent = "Visual"
-
-# Element layout inside a where clause.
-# Values: Vertical | Horizontal | HorizontalVertical | Mixed
-# Default: Vertical
-where_layout = "Mixed"
-
-# Indentation style of a where predicate.
-# Values: Inherit | Tabbed | Visual
-# Default: Visual
-where_pred_indent = "Visual"
-
-# Put a trailing comma on where clauses.
-# Default: false
-where_trailing_comma = true
-
-# Indentation of generics.
-# Values: Inherit | Tabbed | Visual
-# Default: Visual
-generics_indent = "Visual"
-
-# If there is a trailing comma on structs.
-# Values: Always | Never | Vertical
-# Default: Vertical
-struct_trailing_comma = "Vertical"
-
-# If there is a trailing comma on literal structs.
-# Values: Always | Never | Vertical
-# Default: Vertical
-struct_lit_trailing_comma = "Vertical"
-
-# Style of struct definition.
-# Values: Visual | Block
-# Default: Block
-struct_lit_style = "Block"
-
-# Multiline style on literal structs.
-# Values: PreferSingle | ForceMulti
-# Default: PreferSingle
-struct_lit_multiline_style = "PreferSingle"
-
-# Put a trailing comma on enum declarations.
-# Default: true
-enum_trailing_comma = true
-
-# Report all, none or unnumbered occurrences of TODO in source file comments.
-# Values: Always | Unnumbered | Never
-# Default: Never
-report_todo = "Never"
-
-# Report all, none or unnumbered occurrences of FIXME in source file comments.
-# Values: Always | Unnumbered | Never
-# Default: "Never"
-report_fixme = "Never"
-
-# Indent on chain base.
-# Values: Inherit | Tabbed | Visual
-# Default: Tabbed
-chain_base_indent = "Visual"
-
-# Indentation of chain.
-# Values: Inherit | Tabbed | Visual
-# Default: Tabbed
-chain_indent = "Visual"
-
-# Allow last call in method chain to break the line.
-# Default: true
-chains_overflow_last = true
-
-# Reorder import statements alphabetically.
-# Default: false
-reorder_imports = true
-
-# Reorder lists of names in import statements alphabetically.
-# Default: false
 reorder_imported_names = true
```

Comment:
What's the relationship between this configuration and `reorder_imports` that was added in this PR? Just out of curiosity.

Reply:
They have 3 options for reordering imports as I understand it. `reorder_imports` is the base option that needs to be true and handles the `use a; use b;` import ordering, `reorder_imported_names` reorders in`use foo::{a, b, c}` and the 'groups' one reorders blocks of imports that are local vs 3rd party, grouping them together and sorted them in each group.

Reply:
Ah, I see, thanks! The naming of those configurations are...not terribly clear.